v22.1/take-full-and-incremental-backups.md

```diff
@@ -14,10 +14,62 @@ There are two main types of backups:
 
 You can use the [`BACKUP`](backup.html) statement to efficiently back up your cluster's schemas and data to popular cloud services such as AWS S3, Google Cloud Storage, or NFS, and the [`RESTORE`](restore.html) statement to efficiently restore schema and data as necessary. For more information, see [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html).
 
+{% include {{ page.version.version }}/backups/backup-to-deprec.md %}
+
 {{site.data.alerts.callout_success}}
  You can create [schedules for periodic backups](manage-a-backup-schedule.html) in CockroachDB. We recommend using scheduled backups to automate daily backups of your cluster.
 {{site.data.alerts.end}}
 
+## Backup collections
+
+ A _backup collection_ defines a set of backups and their metadata. The collection can contain multiple full backups and their subsequent [incremental backups](#incremental-backups). The path to a backup is created using a date-based naming scheme and stored at the [URI](backup.html#collectionURI-param) passed with the `BACKUP` statement.
+
+There are some specific cases where part of the collection data is stored at a different URI:
+
+- A [locality-aware backup](take-and-restore-locality-aware-backups.html). The backup collection will be stored according to the URIs passed with the `BACKUP` statement: `BACKUP INTO LATEST IN {collectionURI}, {localityURI}, {localityURI}`. Here, the `collectionURI` represents the default locality.
+- As of v22.1, it is possible to store incremental backups at a [different URI](#incremental-backups-with-explicitly-specified-destinations) to the related full backup. This means that one or multiple storage locations can hold one backup collection.
+
+By default, full backups are stored at the root of the collection's URI in a date-based path, and incremental backups are stored in the `/incrementals` directory. The following example shows a backup collection created using these default values, where all backups reside in one storage bucket:
+
+~~~
+Collection:
+|—— 2022
+  |—— 02
+    |—— 09-155340.13/
+      |—— Full backup files
+[...]
+|—— incrementals
```

Comment:
is it worth writing out the full path to the incremental backup files, as you do in the second example? I.e., within `./incrementals`, you have:

```
|—— 2022
  |—— 02
    |—— 25-172907.21/
      |—— 20220325
        |—— 17921.23
          |—— incremental backup files
```

Reply:
I can add this, if you feel it would be clearer